Assets/BossRoom/Scripts/Client/UI/ConnectionStatusMessageUIManager.cs

```diff
@@ -38,17 +42,28 @@ void OnConnectStatus(ConnectStatus status)
                     PopupPanel.ShowPopupPanel("Connection Failed", "The Host is full and cannot accept any additional connections.");
                     break;
                 case ConnectStatus.Success:
+                    m_PopupIdToClose = PopupPanel.ShowPopupPanel("Success!", "Joining Now", false);
```

Comment:
```suggestion
                    m_PopupIdToClose = PopupPanel.ShowPopupPanel("Success!", "Joining Now", isCloseableByUser: false);
```
Just to make this clearer and not have a single bool hanging around